<?php namespace CTIMT\MyOrm\Exception\Platform;

/**
 * Generated by PHPUnit_SkeletonGenerator on 2016-05-26 at 22:48:24.
 */
class BadFunctionCallExceptionTest extends \PHPUnit_Framework_TestCase
{
    /**
     * @expectedException BadFunctionCallException
     * @expectedExceptionMessage someMessage
     */
    public function testBadFunctionCallException()
    {
        throw New BadFunctionCallException('someMessage');
    }
}
